package.json

```diff
@@ -11,7 +11,7 @@
         "configs/"
     ],
     "peerDependencies": {
-        "eslint": "^8.23.0 || >=9.0.0-0"
+        "eslint": ">=8.23.0"
```

Comment:
I think I'd prefer this to be `^8.23.0 || ^9.0.0` as we currently don't know if this will correctly support v10 🤔

Reply:
allowing it will make it a bit easier for eslint to release v10 (eslint itself relies on it, ).

Reply:
We can fix it if we find compat issues  with eslint v10.